proxy/src/graphql/schema.rs

```diff
@@ -620,8 +624,23 @@ impl project::Project {
     fn metadata(&self) -> &project::Metadata {
         &self.metadata
     }
-    fn registered(&self) -> bool {
-        false
+    fn registered(&self) -> Option<ProjectRegistration> {
+        if let Some(registration) = &self.registration {
+            match registration {
+                project::Registration::Org(org_id) => {
+                    Some(ProjectRegistration::Org(OrgRegistration {
+                        org_id: juniper::ID::new(org_id.to_string()),
```

Comment:
Could we return the actual org/user instead of just the id and maybe mock out some sample data?

So far it looks like we'll need:
 - user handle / org name
 - avatarUrl
 - avatarFallback

Reply:
To mock this meaningfully we need orgs first. For the rest we have to decide what behaviour we want as:

* neither user nor orgs have avatar urls
* only orgs have a fallback avatar
* only users maybe have an coco identity attested to them
  * which only maybe is available at the time of query

What we can do for now is to show the `foo / bar` info with a returned avatar from the `avatar` query. How we control that a project appears as registered is open for discussion, it could be a query parameter that we allow for now.

Reply:
> neither user nor orgs have avatar urls

Good point, forgot about that.

> only orgs have a fallback avatar

Why wouldn't a user have a fallback avatar?

> What we can do for now is to show the foo / bar info with a returned avatar from the avatar query. 

Deal.

Reply:
> Why wouldn't a user have a fallback avatar?

AFAIK only identities have fallback avatar, the handle more specifically. @cloudhead and @MeBrei can give more context.

Reply:
I wonder why would we otherwise have user emojis:

https://github.com/radicle-dev/radicle-upstream/blob/e1b7572a0364e81f43ae2541c53663ca82f6000b/proxy/src/avatar.rs#L44:L51

Reply:
I wonder why would we otherwise have user emojis:

https://github.com/radicle-dev/radicle-upstream/blob/master/proxy/src/avatar.rs#L52-L59

Reply:
I guess this will be answered in: https://github.com/radicle-dev/radicle-upstream/issues/230.

Reply:
@rudolfs We have user emojis for the identity handles.

Reply:
Expanded on in https://github.com/radicle-dev/radicle-upstream/issues/230#issuecomment-600516204